build-system/tasks/visual-diff.js

```diff
@@ -456,16 +456,10 @@ async function snapshotWebpages(percy, page, webpages, config) {
         webpage.loading_incomplete_css, webpage.loading_complete_css);
 
     if (webpage.loading_complete_delay_ms) {
-      if (typeof webpage.loading_complete_delay_ms !== 'number' &&
-          webpage.loading_complete_delay_ms > 0) {
-        log('verbose', 'Waiting',
-            colors.cyan(webpage.loading_complete_delay_ms + 'ms'),
-            'for loading to complete');
-        await sleep(webpage.loading_complete_delay_ms || 0);
-      } else {
-        log('warning', 'Skipping unknown delay',
-            webpage.loading_complete_delay_ms);
-      }
+      log('verbose', 'Waiting',
+          colors.cyan(`${webpage.loading_complete_delay_ms}ms`),
+          'for loading to complete');
+      await sleep(webpage.loading_complete_delay_ms);
```

Comment:
Curious, what was the type in the failure case?

Reply:
'number'. I just realized that the conditional was reversed, and that's what caused the problems!
i.e.,
```javascript
if (variable is NOT a number AND variable > 0) {
  sleep(variable);
} else {
  "skipping unknown..."
}
```

I'm surprised these all worked :laughing:

/to @newmuis

Reply:
😑 

I'm not sure it did.  We still had issues that I thought must be independent of this, because I assumed I had the ability to write five lines of code without screwing it up.

Thank you for catching this!